bot/extensions/ping.py

```diff
@@ -0,0 +1,14 @@
+from discord.ext.commands import Cog, command
+from bot import CONFIG
+
+
+class Ping(Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @command(name='ping', help='Send pong', usage=f'{CONFIG.bot.prefix}pong')
```

Comment:
The command does more than simply `Send pong`. You could add that it also check and display the latency of the bot.